lib/libutee/include/tee_internal_api_extensions.h

```diff
@@ -62,4 +63,15 @@ TEE_Result tee_unmap(void *buf, size_t len);
  */
 TEE_Result tee_uuid_from_str(TEE_UUID *uuid, const char *s);
 
+/*
+ * tee_invoke_supp_plugin() - invoke a tee-supplicant's plugin
+ * @uuid:       uuid of the plugin
+ * @cmd:        command for the plugin
+ * @sub_cmd:    subcommand for the plugin
+ * @buf:        data [for/from] the plugin [in/out]
+ * @len:        length of the buf
```

Comment:
Should this be an out parameter too, in case the size is smaller when returning some result?

Reply:
I think yes.
We can replace `size_t len` to `size_t *len` and use one pointer to input and output length.